pymc/sampling/mcmc.py

```diff
@@ -503,7 +613,7 @@ def sample(
     )
 
     sample_args = {
-        "draws": draws,
+        "draws": draws + tune,  # FIXME: Why is tune added to draws?
```

Comment:
probably a relic of old signatures

Reply:
I will open an issue after this PR to investigate it